.gitmodules

```diff
@@ -14,3 +14,10 @@
 	path = packages/protocol/lib/forge-std
 	url = https://github.com/foundry-rs/forge-std
 	branch = chore/v1.5.1
```

Comment:
tag: v1.5.1

Reply:
Can you try to incorporate them (and test) ? I suck so much today morning and yesterday with submodules 😄